src/Form/index.tsx

```diff
@@ -1748,6 +1751,58 @@ function Form({
         };
         updateFieldValues(newValues, true, false);
         client.submitCustom(newValues);
+      } else if (type === ACTION_TELESIGN_SILENT_VERIFICATION) {
+        const phoneNum = fieldValues[
+          action.telesign_target_field_key
+        ] as string;
+        if (validators.phone(phoneNum)) {
+          try {
+            const silentVeriResult: boolean =
+              await client.telesignSilentVerification(phoneNum);
+
+            // set specified field value to the result
+            const key = action.telesign_status_field_key;
+            const newValues = {
+              [key]: silentVeriResult
+            };
+            updateFieldValues(newValues, true, false);
+            client.submitCustom(newValues);
+          } catch (e) {
+            setElementError((e as Error).message);
+            break;
+          }
+        } else {
+          setElementError('Your phone number is invalid or requested too much');
+          break;
+        }
+      } else if (type === ACTION_TELESIGN_VOICE_OTP) {
+        const phoneNum = fieldValues[
+          action.telesign_target_field_key
+        ] as string;
+        if (validators.phone(phoneNum)) {
+          try {
+            await client.telesignVoiceOTP(phoneNum);
+          } catch (e) {
+            setElementError((e as Error).message);
+            break;
+          }
+        } else {
+          setElementError('Your phone number is invalid or requested too much');
```

Comment:
if it reaches here, it failed the validator which means it's only invalid, not requested too much